aiida/cmdline/commands/cmd_group.py

```diff
@@ -353,3 +353,66 @@ def group_copy(source_group, destination_group):
     # Copy nodes
     dest_group.add_nodes(list(source_group.nodes))
     echo.echo_success('Nodes copied from group<{}> to group<{}>'.format(source_group.label, dest_group.label))
+
+
+@verdi_group.group('path')
+def verdi_group_path():
+    """Inspect groups of nodes, with delimited label paths."""
+
+
+@verdi_group_path.command('ls')
+@click.argument('path', type=click.STRING, required=False)
+@click.option('-R', '--recursive', is_flag=True, default=False, help='Recursively list sub-paths encountered')
+@click.option('-l', '--long', 'as_table', is_flag=True, default=False, help='List in long format')
+@click.option(
+    '-d', '--with-description', 'with_description', is_flag=True, default=False, help='Show also the group description'
+)
+@click.option(
+    '-g', '--groups-only', 'groups_only', is_flag=True, default=False, help='Only show paths that contain a group'
+)
+@click.option(
+    '-t',
+    '--type',
+    'group_type',
+    type=types.LazyChoice(valid_group_type_strings),
+    default=user_defined_group,
+    help='Show groups of a specific type, instead of user-defined groups. Start with semicolumn if you want to '
+    'specify aiida-internal type'
+)
+@with_dbenv()
+def group_path_ls(path, recursive, as_table, groups_only, group_type, with_description):
+    """Show a list of existing group paths."""
+    from aiida.tools.groups.grouppaths import GroupPath, InvalidPath
+
+    try:
+        path = GroupPath(path or '', type_string=group_type)
```

Comment:
So, this is just a question from ignorance, but how does that "or" within arguments work? Perhaps I'm misunderstanding something but I think I never saw it before.

Reply:
This is something that @ltalirz is fond of lol. It just means that if `path` evaluates to False, use `''`, and is used here as a shorthand for `'' if path is None else path`